platform/broadcom/sonic-platform-modules-juniper/README.md

```diff
@@ -1 +1,291 @@
+	3) Select "UEFI: IP4 Broadcom NetXtreme Gigabit Ethernet"
+
+## Related Documentation for ONIE:
+
+The following links provide more information about ONIE:
+
+	1. ONIE documentation: https://opencomputeproject.github.io/onie/. 
+	2. How to build and install ONIE on QFX5210-64C-S switch, see 		https://github.com/opencomputeproject/onie/blob/master/machine/juniper/juniper_qfx5210/INSTALL. 
+
+## SONiC Build Process:
+
+The instruction on how to build an ONIE compatible network operating system (NOS) installer image for Juniper Networks QFX5210-64C-S switch, and how to build docker images running inside the NOS is available at https://github.com/Azure/sonic-buildimage#usage. 
+
+
+## Install SONiC on the Juniper Networks QFX5210-64C-S switch:
+
+You need to copy the SONiC image sonic-broadcom.bin to the Juniper Networks QFX5210-64C-S switch. You can copy the sonic-broadcom.bin to an USB memory device and insert it to the USB port of the QFX5210-64C-S switch. You can also use the 'scp' command to copy the sonic-broadcom.bin image to the QFX5210-64C-S switch over the network.
+
+Note: Unmount the USB memory device after copying the sonic-broadcom.bin. For example, umount /dev/sdX, where X is the name of the drive of the USB memory device.
+
+Run the following command to install SONIC:
+   
+```
+For example, 
+ONIE:/var/tmp # onie-nos-install /var/tmp/sonic-broadcom.bin
+```
+
+## Booting SONiC 
+
+The QFX5210-64C-S switch restarts automatically after the SONiC image has been successfully installed. 
+
+1) Select SONiC from the GRUB boot manager. 
+
+                      GNU GRUB  version 2.02
+
+ +----------------------------------------------------------------------------+
+ |*SONiC-OS-master.0-dirty-20190913.060138                                    | 
+ | ONIE                                                                       |
+ |                                                                            |
+ |                                                                            |
+ |                                                                            |
+ |                                                                            |
+ |                                                                            |
+ |                                                                            |
+ |                                                                            |
+ |                                                                            |
+ |                                                                            |
+ |                                                                            | 
+ +----------------------------------------------------------------------------+
+
+      Use the ^ and v keys to select which entry is highlighted.          
+      Press enter to boot the selected OS, `e' to edit the commands       
+      before booting or `c' for a command-line. 
```

Comment:
Surround this in triple backticks to ensure it is displayed in monospaced font.

Reply:
@jleveque 

Please take a look